modules/openapi-generator/src/main/resources/powershell-experimental/http_signature_auth.mustache

```diff
@@ -0,0 +1,162 @@
+{{>partial_header}}
+<#
+.SYNOPSIS
+Get the API key Id and API key file path.
+
+.DESCRIPTION
+Get the API key Id and API key file path. If no api prefix is provided then it use default api key prefix 'Signature'
```

Comment:
It would be good to document which of the methods are public versus which one are for internal use.